runtime/src/bank/builtins/core_bpf_migration/mod.rs

```diff
@@ -416,16 +412,18 @@ mod tests {
             // It should exactly match the original, including upgrade authority
             // and slot.
             let program_data_account_state_metadata: UpgradeableLoaderState =
-                bincode::deserialize(&program_data_account.data()[..PROGRAM_DATA_OFFSET]).unwrap();
+                bincode::deserialize(program_data_account.data()).unwrap();
             assert_eq!(
                 program_data_account_state_metadata,
                 UpgradeableLoaderState::ProgramData {
                     slot: bank.slot, // _Not_ the original deployment slot
                     upgrade_authority_address: self.upgrade_authority_address  // Preserved
                 },
             );
+            let program_data_offset =
```

Comment:
`program_data_offset` should be reverted back to `PROGRAM_DATA_OFFSET` everywhere.

Reply:
I've added a commit for this. Now it's using the constants for both offsets everywhere, including tests.